src/commonMain/kotlin/com/configcat/Config.kt

```diff
@@ -32,6 +31,7 @@ internal data class Entry(
             if (!DateTimeUtils.isValidDate(fetchTimeRaw)) {
                 throw IllegalArgumentException("Invalid fetch time: $fetchTimeRaw")
             }
+            val fetchTimeUnixMillis = fetchTimeRaw.toLong()
             val eTag = cacheValue.substring(fetchTimeIndex + 1, eTagIndex)
             if (eTag.isEmpty()) {
                 throw IllegalArgumentException("Empty eTag value.")
```

Comment:
Please use `require()` everywhere instead of directly throwing `IllegalArgumentException` as `detekt` suggests.